.github/workflows/verify-pr-commit.yml

```diff
@@ -198,8 +198,10 @@ jobs:
     steps:
       - name: Check Out Repo
         uses: actions/checkout@v4
+      - name: Install Nightly Toolchain
+        run: rustup toolchain install nightly-2023-07-13
```

Comment:
This was added here, but not to the release workflow. Is it needed in the release workflow as well?

Reply:
Anywhere the `nightly` is used, which is for `clippy, fmt, doc`, the toolchain would need to be updated.
I tried to add it to the base image [tools/ci/docker/ci-base-image.dockerfile](https://github.com/LibertyDSNP/frequency/pull/1700/files#diff-a02f45eb9e94e21046b873a4e1c2ba8d33ecdfa11468bd4ea52ef6412d23c1c9)
However, it still failed and got added here.
I'll ask Dmitri why that might occur, maybe the docker wasn't updated correctly and these 'rustup' commands can be removed from these actions.

Reply:
New ci-base-image published but it did not fix the redundant commands. 
Writing a new issue to track this down before the next release.